app/src/main/java/com/ucmobiledevelopment/freeways/MainViewModel.kt

```diff
@@ -13,14 +13,13 @@ import com.ucmobiledevelopment.freeways.service.IncidentService
 import kotlinx.coroutines.launch
 
 class MainViewModel(var incidentService : IIncidentService = IncidentService()) : ViewModel(){
-    internal val NEW_INCIDENT = "New Incident"
+    private val NEW_INCIDENT = "New Incident"
```

Comment:
Changed internal -> private as it is only being used here and is deeply immutable. If you plan on using NEW_INCIDENT somewhere else, ignore this.